app/components/visualizer/main-viz/templates/chart.jsx

```diff
@@ -0,0 +1,48 @@
+import D3Chart from "./D3Chart.js"
+
+class Chart extends React.Component {
+  componentDidMount() {
+    let el = React.findDOMNode(this);
+    this.chart = new D3Chart(el, {
+      width: "100%",
```

Comment:
Ok so I had to `git rebase --abort` - I identified your commits and deleted them, but for whatever reason the rebase fails me halfway through and leaves me in some really old state.  I may not understand rebase enough to work my way through it.